modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,103 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install the logging add-on service
+
+{product-tite} (ROSA) provides logging through the `cluster-logging-operator` add-on. This add-on service offers an optional application log forwarding solution based on AWS CloudWatch. This logging solution can be installed after the ROSA cluster is provisioned.
+
+.Procedure
+
+. Enter the following command:
++
+[source,terminal]
+----
+$ rosa install addon cluster-logging-operator --cluster=<cluster_name> --interactive
+----
++
+For `<cluster_name>`, enter the name of your cluster.
+
+. When prompted, accept the default `yes` to install the `cluster-logging-operator`.
+. When prompted, accept the default `yes` to install the optional Amazon CloudWatch log forwarding add-on or enter `no` to decline the installation of this add-on.
+. For the collection of applications, infrastructure, and audit logs, accept the default values or change them as needed:
++
+* *Applications logs*: Lets the Operator collect application logs, which includes everything that is _not_ deployed in the `openshift-*` namespaces. Default: `yes`
+* *Infrastructure logs*: Lets the Operator collect logs from OpenShift Container Platform, Kubernetes, and some nodes. Default: `yes`
+* *Audit logs*: Type `yes` to let the Operator collect node logs related to security audits. By default, Red Hat stores audit logs outside the cluster through a separate mechanism that does not rely on the Cluster Logging Operator. For more information about default audit logging, see the ROSA Service Definition. Default: `no`
+
+. If you are installing the Amazon CloudWatch add-on, use the default log grouping type or change it. Valid values: `logType`, `namespaceName`, or `namespaceUUID`. Default: `logType`
```

Comment:
'grouping type' is not an optional value now.  Please remove this line

Reply:
Done.